src/IO/H5/Python/IterElements.py

```diff
@@ -38,6 +84,14 @@ def iter_elements(volfiles: Union[spectre_h5.H5Vol,
         volfiles = [volfiles]
     for volfile in volfiles:
         dim = volfile.get_dimension()
+        domain = deserialize_domain[dim](volfile.get_domain(obs_id))
+        if domain.is_time_dependent():
+            time = volfile.get_observation_value(obs_id)
+            functions_of_time = deserialize_functions_of_time(
+                volfile.get_functions_of_time(obs_id))
+        else:
+            time = None
+            functions_of_time = None
         all_grid_names = volfile.get_grid_names(obs_id)
         all_element_ids = list(map(ElementId[dim], all_grid_names))
```

Comment:
can you change this to `all_element_ids = [ElementId[dim](grid_name for grid_name in all_grid_names]` ?

Reply:
Don't you like the `map`? I think it's a fine way of converting a list of strings into a list of ElementIds, no?

Reply:
From what I read `map` and `filter` are pretty much considered obsolete to create lists in favor of list comprehensions. But I do not care that much.

Reply:
ok changed to list comprehension